error in opening gnome-terminal

## Description
Just giving chromebrew a try.  gemacs is working fine. But gnome-terminal is not opening up.  


If you executed a specific command:
```shell
gnome-terminal
```

Output:
```shell
# Error constructing proxy for org.gnome.Terminal:/org/gnome/Terminal/Factory0: Unknown or unsupported transport “disabled” for address “disabled:”
```

## Versions and system information
1.11.5

ChromeOS: `cat /etc/lsb-release`
```
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=7487557
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={50F3C95B-CA5B-4AF8-87A2-8CD19588BD12}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={50F3C95B-CA5B-4AF8-87A2-8CD19588BD12}
CHROMEOS_RELEASE_BOARD=kukui-signed-mp-v4keys
CHROMEOS_RELEASE_BRANCH_NUMBER=39
CHROMEOS_RELEASE_BUILDER_PATH=kukui-release/R92-13982.39.0
CHROMEOS_RELEASE_BUILD_NUMBER=13982
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=92
CHROMEOS_RELEASE_DESCRIPTION=13982.39.0 (Official Build) beta-channel kukui 
CHROMEOS_RELEASE_KEYSET=mp-v4
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=beta-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=13982.39.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=13982.39.0
```
Model: lenovo duet chromebook

Architecture: aarch64

> ```
> glib-compile-schemas /usr/local/share/glib-2.0/schemas
> gnome-terminal &
> ```
I tried the above steps. The compile fails for me. Have attached the error log:

```
more /usr/local/tmp/crew/gnome_terminal.2021070814
4824.dir/gnome-terminal-3b79354a357970147ae276a02ca2222db98a0d28/builddir/meson-logs/meson-log.txt
Build started at 2021-07-08T20:18:31.106412
Main binary: /usr/local/bin/python3
Build Options: -Dprefix=/usr/local -Dlibdir=/usr/local/lib -Dmandir=/usr
/local/share/man -Dbuildtype=minsize -Db_lto=true -Dstrip=true -Db_pie=t
rue '-Dcpp_args=-flto -fno-stack-protector -U_FORTIFY_SOURCE -fuse-ld=go
ld -pipe' '-Dc_args=-flto -fno-stack-protector -U_FORTIFY_SOURCE -fuse-l
d=gold -pipe' -Dsearch_provider=false -Dnautilus_extension=false -Ddefau
lt_library=both
Python system: Linux
The Meson build system
Version: 0.58.1
Source dir: /usr/local/tmp/crew/gnome_terminal.20210708144824.dir/gnome-
terminal-3b79354a357970147ae276a02ca2222db98a0d28
Build dir: /usr/local/tmp/crew/gnome_terminal.20210708144824.dir/gnome-t
erminal-3b79354a357970147ae276a02ca2222db98a0d28/builddir
Build type: native build
Project name: gnome-terminal
Project version: 3.41.0
Sanity testing C compiler: cc
Is cross compiler: False.
Sanity check compiler command line: cc /usr/local/tmp/crew/gnome_termina
l.20210708144824.dir/gnome-terminal-3b79354a357970147ae276a02ca2222db98a
0d28/builddir/meson-private/sanitycheckc.c -o /usr/local/tmp/crew/gnome_
terminal.20210708144824.dir/gnome-terminal-3b79354a357970147ae276a02ca22
22db98a0d28/builddir/meson-private/sanitycheckc.exe -flto -fno-stack-pro
tector -U_FORTIFY_SOURCE -fuse-ld=gold -pipe -D_FILE_OFFSET_BITS=64
Sanity check compile stdout:

-----
Sanity check compile stderr:
/usr/local/bin/ld.gold: error: cannot open Scrt1.o: No such file or dire
ctory
/usr/local/bin/ld.gold: error: cannot open crti.o: No such file or direc
tory
/usr/local/bin/ld.gold: error: cannot open crtn.o: No such file or direc
tory
/usr/local/bin/ld.gold: error: cannot find -lc
collect2: error: ld returned 1 exit status

--- a/packages/crew_profile_base.rb
+++ b/packages/crew_profile_base.rb
@@ -35,9 +35,9 @@ class Crew_profile_base < Package
       FileUtils.rm './src/env.d/99-custom' if File.exist? "#{CREW_PREFIX}/etc/env.d/99-custom"
       FileUtils.rm './src/profile.d/99-custom' if File.exist? "#{CREW_PREFIX}/etc/profile.d/99-custom"
       # Custom changes should only go in the above two files.
-      #FileUtils.rm './src/env.d/00-locale' if File.exist? "#{CREW_PREFIX}/etc/env.d/00-locale"
-      #FileUtils.rm './src/env.d/01-editor' if File.exist? "#{CREW_PREFIX}/etc/env.d/01-editor"
-      #FileUtils.rm './src/env.d/02-pager' if File.exist? "#{CREW_PREFIX}/etc/env.d/02-pager"
+      # FileUtils.rm './src/env.d/00-locale' if File.exist? "#{CREW_PREFIX}/etc/env.d/00-locale"
+      # FileUtils.rm './src/env.d/01-editor' if File.exist? "#{CREW_PREFIX}/etc/env.d/01-editor"
+      # FileUtils.rm './src/env.d/02-pager' if File.exist? "#{CREW_PREFIX}/etc/env.d/02-pager"
 
       FileUtils.mkdir_p "#{CREW_PREFIX}/etc/"
       FileUtils.cp_r Dir.glob('./src/*'), "#{CREW_PREFIX}/etc/"
@@ -51,11 +51,11 @@ class Crew_profile_base < Package
 
       # Write our rc files
       @crew_rcfile = <<~CREW_PROFILE_EOF
-                # DO NOT DELETE THIS LINE
-                # See #{CREW_PREFIX}/etc/profile for further details
-                source #{CREW_PREFIX}/etc/profile
+        # DO NOT DELETE THIS LINE
+        # See #{CREW_PREFIX}/etc/profile for further details
+        source #{CREW_PREFIX}/etc/profile
 
-                # Put your stuff under this comment
+        # Put your stuff under this comment
 
       CREW_PROFILE_EOF
       # Must write directly to HOME and not CREW_DEST_HOME to prevent chromebrew from removing ~/.bashrc during reinstall
@@ -66,6 +66,7 @@ class Crew_profile_base < Package
         end
       else
         IO.write("#{HOME}/.bashrc", @crew_rcfile)
+      end
       if File.exist? "#{HOME}/.zshrc"
         if `grep -c "#{@_str}" #{HOME}/.zshrc`.to_i.zero?
           old_zshrc = IO.read "#{HOME}/.zshrc"

--- a/packages/crew_profile_base.rb
+++ b/packages/crew_profile_base.rb
@@ -3,23 +3,12 @@ require 'package'
 class Crew_profile_base < Package
   description 'Crew-profile-base sets up Chromebrew\'s environment capabilities.'
   homepage 'https://github.com/chromebrew/crew-profile-base'
-  version '0.0.1'
+  version '0.0.2'
   license 'GPL-3+'
   compatibility 'all'
   source_url 'SKIP'
 
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/crew_profile_base/0.0.1-pre6_armv7l/crew_profile_base-0.0.1-pre6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/crew_profile_base/0.0.1-pre6_armv7l/crew_profile_base-0.0.1-pre6-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/crew_profile_base/0.0.1-pre6_i686/crew_profile_base-0.0.1-pre6-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/crew_profile_base/0.0.1-pre6_x86_64/crew_profile_base-0.0.1-pre6-chromeos-x86_64.tar.xz'
-  })
-  binary_sha256({
-    aarch64: '042a8b84951511fbc017b2418b8c1eb92d40cba81e1e4e00c0e1d903ad2af032',
-     armv7l: '042a8b84951511fbc017b2418b8c1eb92d40cba81e1e4e00c0e1d903ad2af032',
-       i686: 'c354988e590dbe7cb5d40f683c2a6ec8aecaf497d5844f75bc7a211690ab4406',
-     x86_64: '6ecb01e0cbee6a2bee0b8d86b5fc6d86db92aacf9e2ef2ff892b3d846d5bb4dc'
-  })
+  depends_on 'dbus'
 
   def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
@@ -45,9 +34,10 @@ class Crew_profile_base < Package
       # Don't overwrite custom changes
       FileUtils.rm './src/env.d/99-custom' if File.exist? "#{CREW_PREFIX}/etc/env.d/99-custom"
       FileUtils.rm './src/profile.d/99-custom' if File.exist? "#{CREW_PREFIX}/etc/profile.d/99-custom"
-      FileUtils.rm './src/env.d/00-locale' if File.exist? "#{CREW_PREFIX}/etc/env.d/00-locale"
-      FileUtils.rm './src/env.d/01-editor' if File.exist? "#{CREW_PREFIX}/etc/env.d/01-editor"
-      FileUtils.rm './src/env.d/02-pager' if File.exist? "#{CREW_PREFIX}/etc/env.d/02-pager"
+      # Custom changes should only go in the above two files.
+      #FileUtils.rm './src/env.d/00-locale' if File.exist? "#{CREW_PREFIX}/etc/env.d/00-locale"
+      #FileUtils.rm './src/env.d/01-editor' if File.exist? "#{CREW_PREFIX}/etc/env.d/01-editor"
+      #FileUtils.rm './src/env.d/02-pager' if File.exist? "#{CREW_PREFIX}/etc/env.d/02-pager"
 
       FileUtils.mkdir_p "#{CREW_PREFIX}/etc/"
       FileUtils.cp_r Dir.glob('./src/*'), "#{CREW_PREFIX}/etc/"
@@ -58,10 +48,6 @@ class Crew_profile_base < Package
 
       # Don't overwrite a custom shell rc
       @_str = "source #{CREW_PREFIX}/etc/profile"
-      FileUtils.touch "#{HOME}/.bashrc" unless File.exist? "#{HOME}/.bashrc"
-      FileUtils.touch "#{HOME}/.zshrc" unless File.exist? "#{HOME}/.zshrc"
-      FileUtils.mv "#{HOME}/.bashrc", "#{HOME}/.bashrc.bak" if `grep -c '#{@_str}' #{HOME}/.bashrc`.to_i.zero?
-      FileUtils.mv "#{HOME}/.zshrc", "#{HOME}/.zshrc.bak" if `grep -c '#{@_str}' #{HOME}/.zshrc`.to_i.zero?
 
       # Write our rc files
       @crew_rcfile = <<~CREW_PROFILE_EOF
@@ -70,20 +56,31 @@ class Crew_profile_base < Package
                 source #{CREW_PREFIX}/etc/profile
 
                 # Put your stuff under this comment
+
       CREW_PROFILE_EOF
       # Must write directly to HOME and not CREW_DEST_HOME to prevent chromebrew from removing ~/.bashrc during reinstall
-      IO.write("#{HOME}/.bashrc", @crew_rcfile) unless File.exist? "#{HOME}/.bashrc"
-      IO.write("#{HOME}/.zshrc", @crew_rcfile) unless File.exist? "#{HOME}/.zshrc"
-
-      # Don't overwrite a custom shell rc part 2
-      system "cat #{HOME}/.bashrc.bak >> #{HOME}/.bashrc" if File.exist? "#{HOME}/.bashrc.bak"
-      system "cat #{HOME}/.zshrc.bak >> #{HOME}/.zshrc" if File.exist? "#{HOME}/.zshrc.bak"
+      if File.exist? "#{HOME}/.bashrc"
+        if `grep -c "#{@_str}" #{HOME}/.bashrc`.to_i.zero?
+          old_bashrc = IO.read "#{HOME}/.bashrc"
+          IO.write "#{HOME}/.bashrc", @crew_rcfile + old_bashrc
+        end
+      else
+        IO.write("#{HOME}/.bashrc", @crew_rcfile)
+      if File.exist? "#{HOME}/.zshrc"
+        if `grep -c "#{@_str}" #{HOME}/.zshrc`.to_i.zero?
+          old_zshrc = IO.read "#{HOME}/.zshrc"
+          IO.write "#{HOME}/.zshrc", @crew_rcfile + old_zshrc
+        end
+      else
+        IO.write("#{HOME}/.zshrc", @crew_rcfile)
+      end
 
       puts
       puts 'IT IS IMPERATIVE TO THE FUNCTION OF CHROMEBREW FOR'.lightblue
       puts "#{HOME}/.bashrc to contain the line 'source #{CREW_PREFIX}/etc/profile'.".lightblue
       puts 'This should have happened automatically.'.lightblue
       puts "If the line doesn't exist, add it now and report a bug at https://github.com/chromebrew/crew-profile-base/issues.".lightblue
+      puts "Please put customizations in #{CREW_PREFIX}/etc/env.d/99-custom and #{CREW_PREFIX}/etc/profile.d/99-custom .".lightblue
       puts
     end
   end

--- a/packages/crew_profile_base.rb
+++ b/packages/crew_profile_base.rb
@@ -75,10 +75,6 @@ class Crew_profile_base < Package
       else
         IO.write("#{HOME}/.zshrc", @crew_rcfile)
       end
-
-      puts
-      puts "Please put shell environment customizations in #{CREW_PREFIX}/etc/env.d/99-custom and #{CREW_PREFIX}/etc/profile.d/99-custom .".lightblue
-      puts
     end
   end
 end

--- a/packages/crew_profile_base.rb
+++ b/packages/crew_profile_base.rb
@@ -77,11 +77,7 @@ class Crew_profile_base < Package
       end
 
       puts
-      puts 'IT IS IMPERATIVE TO THE FUNCTION OF CHROMEBREW FOR'.lightblue
-      puts "#{HOME}/.bashrc to contain the line 'source #{CREW_PREFIX}/etc/profile'.".lightblue
-      puts 'This should have happened automatically.'.lightblue
-      puts "If the line doesn't exist, add it now and report a bug at https://github.com/chromebrew/crew-profile-base/issues.".lightblue
-      puts "Please put customizations in #{CREW_PREFIX}/etc/env.d/99-custom and #{CREW_PREFIX}/etc/profile.d/99-custom .".lightblue
+      puts "Please put shell environment customizations in #{CREW_PREFIX}/etc/env.d/99-custom and #{CREW_PREFIX}/etc/profile.d/99-custom .".lightblue
       puts
     end
   end

